app/views/layouts/_navbar.html.erb

```diff
@@ -15,17 +23,17 @@
           <a class="nav-link" href="#">Orders</a>
         </li>
         <li class="nav-item">
-          <a class="nav-link">Pay</a>
+          <a class="nav-link" href="#">Pay</a>
         </li>
       </ul>
-      <form class="d-flex" role="search">
-        <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
-        <button class="btn btn-outline-light" type="submit">Search</button>
-      </form>
-      <button type="button" class="btn btn-outline-light">
-        <i class="bi bi-cart-fill"></i>
-        Cart
-      </button>
+      <div class="d-flex align-items-center">
+      <% if session[:user_id] %> 
+        <%= button_to "Logout", logout_path, method: :delete, class: "btn btn-outline-secondary me-3" %>  
```

Comment:
Show here logged_in as `user name`.
we can add a button to users profile as well.

Reply:
Sir will do this in another PR.